oozie/oozie.sh

```diff
@@ -27,14 +27,15 @@ set -euxo pipefail
 
 # Use Python from /usr/bin instead of /opt/conda.
 export PATH=/usr/bin:$PATH
+export DATAPROC_IMAGE=$(/usr/share/google/get_metadata_value image)
 
-function retry_apt_command() {
+function retry_command() {
   local cmd="$1"
   for ((i = 0; i < 10; i++)); do
     if eval "$cmd"; then
       return 0
     fi
-    sleep 5
+    sleep $((i * 5 ))
```

Comment:
Revert, this will make backoff exponential, which we probably do not want to do.